src/sidebar.ts

```diff
@@ -64,103 +64,132 @@ const openIdSSO: SidebarItemConfig = {
   ],
 }
 
-const referenceSidebar: ExtendSidebar = [
+const api: ExtendSidebar = [
   homeLink,
+  "reference/api",
+
+  "ecosystem/api-design",
   {
     type: "category",
-    label: "Reference",
+    label: "Operations",
     collapsed: false,
     collapsible: false,
+    link: {
+      type: "generated-index",
+      slug: "/category/operations-reference",
+    },
     items: [
-      "reference/api",
-      "sdk",
-      "ecosystem/api-design",
-      {
-        type: "category",
-        label: "Operations reference",
-        collapsed: false,
-        collapsible: false,
-        link: {
-          type: "generated-index",
-        },
-        items: [
-          "concepts/personal-access-token",
-          "guides/cors",
-          "guides/custom-domains",
-          "guides/api-rest-pagination",
-          "guides/workspaces",
-          "guides/rate-limits",
-          "guides/ip-allowlist",
-          "api/eventual-consistency",
-          "console/usage-billing",
-        ],
-      },
+      "concepts/personal-access-token",
+      "guides/cors",
+      "guides/custom-domains",
+      "guides/api-rest-pagination",
+      "guides/workspaces",
+      "guides/rate-limits",
+      "guides/ip-allowlist",
+      "api/eventual-consistency",
+      "console/usage-billing",
+    ],
+  },
+]
+
+const sdk: ExtendSidebar = [
+  homeLink,
+  "sdk",
+  {
+    type: "category",
+    label: "Identity SDKs",
+    collapsed: false,
+    collapsible: false,
+    link: {
+      type: "doc",
+      id: "kratos/sdk/overview",
+    },
+    items: ["kratos/self-hosted/go"],
+  },
+  {
+    type: "category",
+    label: "OAuth2 SDKs",
+    collapsed: false,
+    collapsible: false,
+    link: {
+      type: "doc",
+      id: "hydra/sdk/overview",
+    },
+    items: ["hydra/self-hosted/go", "hydra/self-hosted/js"],
+  },
+  {
+    type: "category",
+    label: "Permission SDKs",
+    collapsed: false,
+    collapsible: false,
+    link: {
+      type: "doc",
+      id: "keto/sdk/overview",
+    },
+    items: ["keto/self-hosted/go", "keto/sdk/python"],
+  },
+]
+const cli: ExtendSidebar = [
+  homeLink,
+  {
+    type: "category",
+    label: "Ory CLI reference",
```

Comment:
what if we add CLI to product section?

Reply:
I see why it might fit under products. I left Product section to represent Ory core pillar products but we can think about it further and reconsider.